helm/defectdojo/templates/django-deployment.yaml

```diff
@@ -51,7 +51,7 @@ spec:
       {{- end }}
       {{- if .Values.django.mediaPersistentVolume.enabled }}
       securityContext:
-        fsGroup: {{ .Values.django.mediaPersistentVolume.fsGroup | default 1001 }}
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
```

Comment:
Should this only be included when `django.mediaPersistentVolume.enabled` is true? Seems like that's more relevant for the specific `fsGroup` key that was previously being set on its own.